lib/web_ui/dev/mobile_unit_test_manager.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'driver_runner.dart';
+import 'environment.dart';
+import 'utils.dart';
+
+/// Unit test manager for IOS Safari and Chrome on Android.
+class MobileUnitTestManager {
+  final String _browser;
+
+  MobileUnitTestManager(this._browser);
+
+  Future<void> runUnitTests() async {
+    // TODO(nurhan): Add Chrome on Android.
+    // https://github.com/flutter/flutter/issues/53693
+    assert(_browser == 'ios-safari');
```

Comment:
Does `skip` work the same way it does for other browsers? What would one need to do to skip a test only on mobile Safari?